CK2Plus/common/cultures/00_latin.txt

```diff
@@ -165,4 +165,77 @@ latin = {
 		mat_grm_name_chance = 60
 		mother_name_chance = 10
 	}
+
+	norman = {
```

Comment:
Norman should not be in the Latin group. It belongs in either gallo_roman (where it is currently) or north_germanic

Reply:
I have added the new names to the Gallo-Roman group, and undone my commits to the Latin group.